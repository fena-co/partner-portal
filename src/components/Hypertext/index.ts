import styled from 'styled-components';

const Hypertext = styled.a`
  font-family: Montserrat;
  font-style: normal;
  font-weight: 500;
  font-size: 14px;
  line-height: 17px;
  color: #38b6ff;
  cursor: pointer;
  margin: 0;
  &:hover {
    text-decoration-line: underline;
  }
  &:active {
    text-decoration-line: underline;
    color: #6c6c8a;
  }
`;

export default Hypertext;
