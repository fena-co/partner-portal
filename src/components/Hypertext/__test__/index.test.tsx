import Hypertext from '../index';
import renderer from 'react-test-renderer';

describe('Hypertext Block', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Hypertext />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
