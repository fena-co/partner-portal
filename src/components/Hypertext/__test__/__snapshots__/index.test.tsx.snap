// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hypertext Block renders correctly 1`] = `
.c0 {
  font-family: Montserrat;
  font-style: normal;
  font-weight: 500;
  font-size: 14px;
  line-height: 17px;
  color: #38b6ff;
  cursor: pointer;
  margin: 0;
}

.c0:hover {
  -webkit-text-decoration-line: underline;
  text-decoration-line: underline;
}

.c0:active {
  -webkit-text-decoration-line: underline;
  text-decoration-line: underline;
  color: #6c6c8a;
}

<a
  className="c0"
/>
`;
